### DISCUSSION TITLE
Importing a font via css-loader gives "file://"

### DISCUSSION THREAD

**Discussion by alverdal** [webpack/webpack#15039] [Q&A]
We are importing fonts and processing them via webpack (v5.46.0), 'vue-style-loader' (v) and 'css-loader' (v6.2.0).
The resulting path to font that is sent to the browser is "url(file:///mf/fonts/SharpSansNo1-Book.2ee0a.woff2)" and the browser complaints that it cannot fetch this font (which makes sense).
However, if we set css-loader option "esModule=false" we get a correct path to the font "url(/mf/fonts/SharpSansNo1-Book.2ee0a.woff2)". (or downgrade to v5 version of css-loader also seem to solve this issue) 

Anybody who can advice on why we have to use esModule=false to get correct urls sent to the browser?

in stylesheet:
@font-face {
	font-family: 'SharpSansNo1Book';
	font-display: swap;
	src: url('../../../../assets/fonts/SharpSansNo1_Books/SharpSansNo1-Book.woff2') format('woff2') 
}

results in browser injected style-sheet in html head: 
@font-face{font-display:swap;font-family:SharpSansNo1Book;font-style:normal;font-weight:400;src:url(file:///mf/fonts/SharpSansNo1-Book.2ee0a.woff2)

**Comment by alexander-akait** [comment]
It means you try to extract CSS and don't provide public path

**Reply by alverdal** [reply]
I do have provided a "publicPath" in the webpack config as '/' which seem to be added to the path, three '///'.

**Reply by alexander-akait** [reply]
If you create reproducible test repo I will show how to fix you

**Reply by alverdal** [reply]
I believe that the issue is related to that the package that we're importing is not exported as a es6 module and that's why I need to use esModule=false .

**Reply by alexander-akait** [reply]
If you have `file://`, it means something wrong in configuration with CSS
